src/mitol/payment_gateway/api.py

```diff
@@ -361,6 +433,106 @@ def prepare_checkout(
             "method": "POST",
         }
 
+    def get_client_configuration(self):
+        """
+        Get the configuration required for CyberSource API client
+        """
+        configuration_dictionary = {
+            "authentication_type": "http_signature",
+            "merchantid": settings.MITOL_PAYMENT_GATEWAY_CYBERSOURCE_MERCHANT_ID,
+            "run_environment": CYBERSOURCE_LIVE_API_URL
+            if settings.ENVIRONMENT == "prod"
+            else CYBERSOURCE_TEST_API_URL,
```

Comment:
Instead of conditioning this on `ENVIRONMENT` it would be more flexible to use a dedicated environment variable.

Could you add a settings (e.g. `cybersource.py`) file for `MITOL_PAYMENT_GATEWAY_*` settings and add that there? Also if you don't mind, it would be great to move over the existing settings that are defined in mitx online here: https://github.com/mitodl/mitxonline/blob/main/main/settings.py#L971-L1000

You can use [this](https://github.com/mitodl/ol-django/tree/main/src/mitol/mail/settings) as a reference for how to go about it.

Reply:
Addressed